srcpkgs/meson/patches/10702.patch

```diff
@@ -1,59 +0,0 @@
-From 869b95d0c2a854d73db0fceb7d0dbee9dc88cf46 Mon Sep 17 00:00:00 2001
-From: Eli Schwartz <eschwartz@archlinux.org>
-Date: Tue, 16 Aug 2022 22:42:53 -0400
-Subject: [PATCH] minstall: do not trample install_mode by rpath fixer
-
-install_mode can include the setuid bit, which has the special property
-(mentioned in the set_mode logic for minstall itself) of needing to come
-last, because it "will get wiped by chmod" (or at least chown).
-
-In fact, it's not just chown that wipes setuid, but other changes as
-well, such as the file contents. This is not an issue for install_data /
-custom_target, but for compiled outputs, we run depfixer to handle
-rpaths. This may or may not cause edits to the binary, depending on
-whether we have a build rpath to wipe, or an install rpath to add. (We
-also may run `strip`, but that external program already has its own mode
-restoration logic.)
-
-Fix this by switching the order of operations around, so that setting
-the permissions happens last.
-
-Fixes https://github.com/void-linux/void-packages/issues/38682
```

Comment:
This patch was introduced after Meson 0.63.1 was released, and is a backport of https://github.com/mesonbuild/meson/pull/10702 which is milestoned for 0.63.2

We do plan to have a .2 release soon (it's even queued for tagging, but some macOS CI job mysteriously doesn't have a test dependency despite installing it).

Reply:
fixed